pkg/v2/tkr/util/sets/compatibility.go

```diff
@@ -1,14 +1,12 @@
 // Copyright 2022 VMware, Inc. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package version
+package sets
```

Comment:
I'd like to move this package (`sets`) to `apis/run/util` and keep this `Compatibility` interface in the `version` package.

Reply:
Sure, let me look at making this change.